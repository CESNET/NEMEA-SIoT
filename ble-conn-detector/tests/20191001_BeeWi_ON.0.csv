time TIMESTAMP,macaddr DEV_ADDR,uint8 ATYPE,int8 RSSI
2033-01-09T06:14:56.000,07:00:00:cc:06:ad,32,118
2019-09-25T10:23:00.588,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:00.698,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:00.804,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:00.906,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:01.009,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:01.114,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:01.218,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:01.322,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:01.536,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:01.641,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:01.746,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:01.854,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:01.956,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:02.064,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:02.284,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:02.496,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:02.601,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:02.707,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:02.810,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:02.913,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:03.124,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:03.232,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:03.341,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:03.440,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:03.549,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:03.651,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:03.754,6d:73:5a:f3:7b:98,0,-41
2019-09-25T10:23:03.863,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:03.966,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:04.068,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:04.177,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:04.284,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:23:04.498,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:04.708,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:04.814,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:04.920,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:05.029,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:05.131,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:05.238,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:05.339,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:05.551,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:05.763,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:05.868,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:05.979,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.081,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.190,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.289,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:06.399,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.504,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:06.607,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.708,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.816,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:06.923,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.028,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.130,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:07.240,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.347,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:07.453,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.554,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:07.661,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.769,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:07.871,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:07.975,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:08.079,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:08.178,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:08.285,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:08.388,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:08.498,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:08.606,6d:73:5a:f3:7b:98,0,-41
2019-09-25T10:23:08.707,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:08.814,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:23:08.916,6d:73:5a:f3:7b:98,0,-40
2019-09-25T10:23:09.025,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:23:09.132,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:09.241,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:09.345,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:12.777,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:12.994,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.204,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.307,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.413,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.519,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.621,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.722,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:13.829,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:13.936,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.041,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:14.141,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.243,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.352,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:14.457,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.563,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.668,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.771,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:14.880,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:14.982,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:15.091,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.199,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.302,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.408,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.510,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.619,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:15.720,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:15.824,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:15.928,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:16.135,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:16.347,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:16.447,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:16.547,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:16.651,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:16.861,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:16.961,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:17.066,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:17.173,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:17.272,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:17.382,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:17.490,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:17.594,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:17.698,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:17.801,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:17.910,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:18.014,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:23:18.119,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:18.225,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:23:18.324,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:23:18.430,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:23:18.531,6d:73:5a:f3:7b:98,0,-44
