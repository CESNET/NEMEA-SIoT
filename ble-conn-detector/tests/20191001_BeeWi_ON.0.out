2019-09-25T10:23:12.777,6d:73:5a:f3:7b:98,0,3432,0,"The device 6d:73:5a:f3:7b:98 was in use for 3432us."
